scripts/jenkins/Jenkinsfile

```diff
@@ -1,5 +1,25 @@
 platforms = ["centos7", "debian9", "ubuntu16", "ubuntu18"]
 
+def detectChanges() {
+    def string = ""
+    def changeLogSets = currentBuild.changeSets
+    for (int i = 0; i < changeLogSets.size(); i++) {
+        def entries = changeLogSets[i].items
+        for (int j = 0; j < entries.length; j++) {
+            def entry = entries[j]
+            def files = new ArrayList(entry.affectedFiles)
+            for (int k = 0; k < files.size(); k++) {
+                def file = files[k]
+                if (file.path =~ /(^debian\/.*)|(redhat\/freeradius.spec)/) {
+                    echo "file ${file.path} matches"
```

Comment:
Matches what?

Reply:
You know what, that line was just so I could debug it when I was starting out.  I'll pull it.

I also need to sign the commit

Reply:
Or, maybe I'll just add some text to clarify

Reply:
Yes, I do know what cos I'm reading the script, nobody will no what when they're looking at the log output which was of course my point :)

Reply:
fixed now

Reply:
Maybe some comments about what the no-cache flag does, and why we're only applying it to a subset of builds?